stream-chat-android-core/src/main/java/io/getstream/chat/android/models/FilterObject.kt

```diff
@@ -21,31 +21,61 @@ package io.getstream.chat.android.models
  */
 public sealed class FilterObject
 
+@ConsistentCopyVisibility
```

Comment:
What about using the `-Xconsistent-data-class-copy-visibility` compiler flag instead? So that we could change a single line when this feature becomes stable.

Reply:
For reference: https://kotlinlang.org/api/core/kotlin-stdlib/kotlin/-consistent-copy-visibility/